### ISSUE TITLE
Delivery SDK: Rich text parser swallows stacktrace

### ISSUE THREAD

**Issue by andreyrd** [Kentico/kentico-cloud-js#184]
### Brief bug description
Rich text element parser swallows the stacktrace of any error that might happen inside of the rich text resolver function.

### Repro steps
```
 class MyContentItem extends KenticoCloud.ContentItem {
  constructor () {
    super({ richTextResolver: function (item) {
      someFunctionThatThrowsAnError();
    } });
  }
}
```

### Expected behavior

The call to the function that throws an error should be included in the stacktrace, as well as any functions called inside of that one. But instead the last item on the stacktrace is always `Parse5RichTextParser.resolveRichTextField`. The error that was thrown is included, but the stacktrace is lost.

### Test environment

 - Platform/OS: macOS / Node.js 10.15.2
 - Version: 6.0.0-beta.8

### Additional context

The offending line can be [found here](https://github.com/Kentico/kentico-cloud-js/blob/master/packages/delivery/lib/parser/adapters/parse5-rich-text.parser.ts#L51). When the new error is created, any previous stack trace is lost. And now we can't debug where the real error is, because the stacktrace stops at `resolveRichTextField`.

Is it really necessary to create and throw a new error, saying "Parsing HTML failed"? Why even use a try catch? Why not let the actual error make its way up and get caught wherever our application handles errors? This discards and obscures the actual errors that happen.

Alternatively, if creating a new error is necessary, there are ways to preserve the stack: https://stackoverflow.com/questions/42754270/re-throwing-exception-in-nodejs-and-not-losing-stack-trace

**Comment by Enngage** [comment]
Thanks for submitting the issue! You are right, the try/catch is unnecessary here, and so I removed it. It will be included in next beta release.

**Comment by Enngage** [comment]
Released in `In 6.0.0-beta.9`
